modules/ztp-deploying-a-site.adoc

```diff
@@ -91,6 +91,32 @@ The `enable-crun-master.yaml` and `enable-crun-worker.yaml` CR files are in the
 For more information, see "Customizing extra installation manifests in the {ztp} pipeline".
 ====
 
+.. Optional: To reduce the number of policies for a deployment, you can use hub-side templates with a `PolicyGenTemplate`.
+You can use the `spec.clusters.siteConfigMap` section of the `SiteConfig` custom resource (CR) to specify per-site data to be used for hub-side templating.
+The following is an example use of the `data` field in `SiteConfig`:
++
+[source,yaml]
+----
+siteConfigMap:
+   name: site-example-sno <1>
+   namespace: ztp-pgs <2>
+   data:
+      dell-poweredge-xr12-ptpcfgslave-profile-interface: "ens5f0"
+      dell-poweredge-xr12-cpu-isolated: "2-19,22-39"
+      dell-poweredge-xr12-cpu-reserved: "0-1,20-21"
+      dell-poweredge-xr12-hugepages-default: "1G"
+      dell-poweredge-xr12-hugepages-size: "1G"
+      dell-poweredge-xr12-hugepages-count: "32"
```

Comment:
Maybe we can remove this and instead include:
```
sno-1-sriov-network-vlan-1: "140"
sno-1-sriov-network-vlan-2: "150"
```

Reply:
thanks